web/src/lib/components/NostrNoteActions.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
```

Comment:
Time to create a Nostr/ directory?

Reply:
Yes, you're right. Is it ok if I change it in the next PR that you'll receive later/tomorrow?

I'm saying this because the next one is based on this one, so if I change it here, I'll have to do the move and changes to the imports twice.